dockerfile/anaconda-iso-creator/lorax-build-webui.j2

```diff
@@ -38,6 +38,9 @@ REPO_DIR=/tmp/anaconda-rpms/
 # create repo from provided Anaconda RPMs
 mkdir -p $REPO_DIR
 cp -a $INPUT_RPMS/* $REPO_DIR || echo "RPM files can't be copied!"  # We could just do the build with official repositories only
+git clone --depth 1 https://github.com/cockpit-project/bots.git
```

Comment:
Could you please add a comment before this new section and seperate it before and after with a new line? thanks